repose-aggregator/artifacts/filter-bundle/src/config/filters/rate-limiting.cfg.xml

```diff
@@ -1,23 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<!-- http://wiki.openrepose.org/display/REPOSE/Rate+Limiting+Filter -->
-<rate-limiting xmlns="http://docs.openrepose.org/repose/rate-limiting/v1.0">
-    <!--
-        Defines an endpoint with a matching regex to bind GET requests for
-        returning live rate limiting information.
-    -->
-    <request-endpoint uri-regex="/limits" include-absolute-limits="false"/>
-
-    <global-limit-group>
-        <limit id="global" uri="*" uri-regex=".*" value="1000" unit="MINUTE"/>
-    </global-limit-group>
-
-    <!-- Limits for all other requests -->
-    <limit-group id="limited" groups="limited" default="true">
-        <limit id="all" uri="*" uri-regex="/.*" http-methods="POST PUT GET DELETE" unit="MINUTE" value="10"/>
+<rate-limiting datastore="distributed/hash-ring" xmlns="http://docs.openrepose.org/repose/rate-limiting/v1.0">
+    <request-endpoint uri-regex="/service/limits/" include-absolute-limits="true"/>
+    <limit-group id="customer-limits " groups="customer" default="true">
+        <limit id="one" uri="/service/*" uri-regex="/service/([\d^/]*)/.*" http-methods="GET" unit="SECOND" value="1"/>
+        <limit id="two" uri="/service/*" uri-regex="/service/([\d^/]*)/.*" http-methods="DELETE" unit="MINUTE"
+               value="5"/>
+        <limit id="three" uri="/service/*" uri-regex="/service/([\d^/]*)/.*" http-methods="PUT" unit="HOUR" value="15"/>
+        <limit id="four" uri="/service/*" uri-regex="/service/([\d^/]*)/.*" http-methods="POST" unit="DAY" value="2"/>
+    </limit-group>
+    <limit-group id="admin-limits" groups="admin">
+        <limit id="five" uri="/service/admin/*" uri-regex="/service/admin/(.*)" http-methods="GET POST PUT DELETE"
+               unit="SECOND" value="1"/>
     </limit-group>
-
-    <!-- Limits for WhiteListed IPs -->
-    <limit-group id="unlimited" groups="unlimited" default="false"/>
-
```

Comment:
Yes. I defaulted to the working examples since those were all guaranteed to be at least syntactically correct since they were being tested. There were several that were out of sync between the shipping example and the current features.